requirements.txt

```diff
@@ -1 +1,2 @@
 six==1.16.0
+scapy==2.4.5
```

Comment:
It seems that Scapy was removed on purpose in #147.

cc @sborkows 

That being said, I don't think adding it back defeats the purpose of #147, and not having it in requirements.txt may confuse users. However, the README change which gives a separate command to install scapy should be reverted as well if we go this route.

Reply:
The problem that has arisen is that we do not have a ready solution now. Removing Scapi made us lose the "Plug & Play" defacto :(

Reply:
I don't want to confuse it, I will remove it from reaquirements.txt, I will add its manual installation in CI.